circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/replica/Replica.java

```diff
@@ -435,6 +427,9 @@ public ReplicaLocationManager getLocationManager(
       SourceLocationManager sourceLocationManager) {
     CleanupLocationManager cleanupLocationManager = CleanupLocationManagerFactory
         .newInstance(eventId, housekeepingListener, replicaCatalogListener, tableReplication);
+
+    sourceLocationManager.getTableLocation();
```

Comment:
What does this line do? The result isn't being assigned to anything - is it needed?